### ISSUE TITLE
Not remembering search filter after leaving list

### ISSUE THREAD

**Issue by cjessex** [scoumbourdis/grocery-crud#409]
Hi,

I am using datatables theme, and when searching for something in the list, it filters the data fine. If I then click add / edit and return back to the list, the field is still populated with the search (which I want), however, the list is no longer filtered by that search.

**Comment by mckaygerhard** [comment]
this its the expected behaviour and this its a duplication of #410 problem, always will happened with modern browsers.. jajaja

**Comment by cjessex** [comment]
Further information.. when running on a Linux server, with Apache this error doesn't occur, only when running on a Windows machine running XAMPP server.

**Comment by mckaygerhard** [comment]
as always a stupid guindows problem.. so there's no a problem.. this issue must be closed now!

**Comment by charlienetcom** [comment]
No, the same source is running on Apache with same issue. Dropdown is prefilled after data is inserted, but unless explicitly set again, it does not get the value when submitting. Therefore this isn't a windows problem.
